.github/workflows/release_post_release_ceremony.yaml

```diff
@@ -0,0 +1,32 @@
+name: Release - Post Release Ceremony
+"on":
+  release:
+    types: [published]
+  workflow_dispatch:
+
+# The workflow needs the permission to a PR
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  post-release-ceremony:
+    name: Post Release Ceremony
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout TimescaleDB
+        uses: actions/checkout@v4
+
+      - name: Get latest tag
+        run: |
+          git fetch --tags
+          echo $(git describe --tags `git rev-list --tags --max-count=1`)
+
+      - name: Cherry pick last commit from tag to main
+        run: |
+          echo "TODO"
+
+      - name: Create PR for cherry picked commit
+        run: |
+          echo "TODO"
```

Comment:
Since there are no actions here, why do you want to include this workflow in the PR?

Reply:
Yea, I'll remove the post release workflow as a whole and do it in a separate PR

Reply:
resolved via https://github.com/timescale/timescaledb/pull/8282/commits/8d62a237b0fa8e01f96fed016ccbbdb579527aab